docs/_source/guides/llms/practical_guides/fine_tune.md

```diff
@@ -518,149 +796,111 @@ encoding = tokenizer([inputs], return_tensors="pt")
 outputs = model.generate(**encoding, max_new_tokens=30)
 output_text = tokenizer.decode(outputs[0])
 print(output_text)
+# Yes it is, toads are a sub-classification of frogs.
 ```
 
-:::
-
-::::
-
-#### TRLX
+### Direct Preference Optimization
 
-[TRLX](https://github.com/CarperAI/trlx) gives the option to use a `reward function` or a `reward-labeled` dataset in combination with Proximal Policy Optimization (PPO) for the reinforcement learning step, which can be used by defining a PPO policy configuration. During this step, we infer rewards to mimic the human evaluation of generated texts. Additionally, [Hugging Face Accelerate](https://huggingface.co/docs/accelerate/index) can be used to speed up training or [Ray Tune](https://docs.ray.io/en/latest/tune/index.html) to optimize hyperparameter tuning.
+#### Background
 
-```python
-from trlx.data.default_configs import default_ppo_config
+The [TRL](https://huggingface.co/docs/trl) library implements and alternative way to incorporate human feedback into an LLM which is called Direct Preference Optimization (DPO). This approach skips the step of training a separate reward model and directly uses the preference data during training as measure for optimization of human feedback. In order to properly use th
 
-config = default_ppo_config()
-config.model.model_path = 'gpt2'
-config.train.batch_size = 16
+```{note}
+DPO requires a trained supervised fine-tuned model to function. Take a look at that task outline above to train your own model.
 ```
 
-::::{tab-set}
+```{include} /_common/dolly_dataset_info.md
+```
 
-:::{tab-item} reward function
+In case of training an PPO, we then use the prompt and context data and correct the generated response from the SFT model by using the reward model. Hence, we will need to format the following `text`.
```

Comment:
```suggestion
In case of training using PPO, we then use the prompt and context data and correct the generated response from the SFT model by using the reward model. Hence, we will need to format the following `text`.
```